feature/xhr/aggregate/index.js

```diff
@@ -121,7 +121,7 @@ baseEE.on('interactionDiscarded', function (interaction) {
 function prepareHarvest(options) {
   options = options || {}
 
-  if (ajaxEvents.length === 0) {
+  if (ajaxEvents.length === 0 || !allAjaxIsEnabled()) {
```

Comment:
Instead of checking here, I would recommend not starting the harvesting scheduler at all, which is on line 43 - 44.